src/components/TableRow.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import moment from "moment";
+const TableRow = ({ list }) => {
+  const checkIn = list.checkInDate;
+  const checkOut = list.checkOutDate;
+  const duration = moment(checkOut).diff(checkIn);
+  const durationInHours = duration / 1000 / 60 / 60;
+  const durationInDays = Math.round(durationInHours / 24);
```

Comment:
If you look at the Moment documentation, you'll see that the `diff` method actually has a way of calculating this for you: https://momentjs.com/docs/#/displaying/difference/

Reply:
correct,
 const duration = moment(checkOut).diff(checkIn, 'days');